design-documents/frontend/server-side-rendering.md

```diff
@@ -0,0 +1,129 @@
+# Server-Side Rendering (SSR)
+
+_This document refers to SSR in the context of a React web application (e.g., PWA). The concept of SSR does not apply to native apps._
+
+_SSR belongs to the BFF layer of [service isolation](https://github.com/magento/architecture/tree/master/design-documents/service-isolation)._
+
+SSR can be a challenge to implement correctly, but the benefits have the potential to provide the unparalleled performance you seek to decrease bounce rate and drive more business.
+
+## Pros
+
+- Faster, first meaningful paint.
+- No need to render initial content via JavaScript on the client.
+- SEO/bot support.
+- Better SEO performance.
```

Comment:
- `SEO/bot support`
- `Better SEO performance.`

These seem duplicative, maybe just combine to 1